src/pres/loader/Filters.js

```diff
@@ -156,4 +158,4 @@ export default class User extends React.Component {
         </Accordion>
 
     }
-}
\ No newline at end of file
+}
```

Comment:
Looks like I've got line endings configured differently from you -- happy to change these to be in line with what you've got (I'll do that at the end, though, for simplicity)